src/main/scala/sangria/marshalling/circe.scala

```diff
@@ -106,4 +108,53 @@ object circe {
         case Left(error) => throw InputParsingError(Vector(error.getMessage))
       }
     }
+
+  implicit object CirceAnyInputUnmarshaller extends InputUnmarshaller[Json @@ AnySupport] {
```

Comment:
AnySupport is a tag defined in this [PR](https://github.com/sangria-graphql/sangria-marshalling-api/pull/35).